.eslintrc.js

```diff
@@ -6,15 +6,25 @@
 /* eslint-env node */
 
 module.exports = {
-    extends: "ckeditor5",
-    env: {
-        "browser": true,
-        "webextensions": true
-    },
-    rules: {
-        'max-len': 'off'
-    },
-    parserOptions: {
-        ecmaVersion: 2018,
+  extends: [
+    'ckeditor5',
+    'plugin:react/recommended'
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true
     }
+  },
+  env: {
+    mocha: false
+  },
+  settings: {
+    react: {
+      version: '16.3'
+    }
+  },
+  rules: {
+    'react/prop-types': 'off'
```

Comment:
Maybe it's a good idea to add `propTypes` to ensure if a component received props with exact type?

Reply:
And with propTypes, we'd need to introduce the development build as well. propTypes are cut out from the production build.

Reply:
I checked it. `propTypes` works, but we'd need to build code in the development mode and intensively check console for the warnings since we don't have tests for the package. So I'm against it.